index.html

```diff
@@ -91,6 +91,35 @@ <h2 class="page-form__heading">Send Email</h2>
           </div>
         </section> 
       </div>
+      <div class="container-column">
```

Comment:
W sumie można zrobić na odwrót, zamień diva clasy container-column, na section, a a section na diva, po jak już powinno być sekcja>div, a nie div>sekcja to możesz pozmieniać

Reply:
Done